README.md

```diff
@@ -64,6 +78,17 @@ Or you can also define a `jest-junit` key in your `package.json`.  All are **str
 }
 ```
 
+Or you can define your options in your reporter configuration.
+
+```
```

Comment:
can add `js` for that sweet syntax highlighting?

Reply:
On it!